airbyte-integrations/connectors/destination-s3/src/test-integration/java/io/airbyte/integrations/destination/s3/S3CsvAssumeRoleDestinationAcceptanceTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.s3;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.airbyte.cdk.integrations.destination.s3.S3BaseCsvDestinationAcceptanceTest;
+import io.airbyte.cdk.integrations.standardtest.destination.ProtocolVersion;
+import java.util.HashMap;
+import java.util.Map;
+
+public class S3CsvAssumeRoleDestinationAcceptanceTest extends S3BaseCsvDestinationAcceptanceTest {
+
+  @Override
+  public ProtocolVersion getProtocolVersion() {
+    return ProtocolVersion.V1;
```

Comment:
ugggh. does anything break if you set this to v0? (v1 was a thing that should no longer exist, but there's still random remnants everywhere)